versioned/persist/tx/src/main/java/org/projectnessie/versioned/persist/tx/TxDatabaseAdapter.java

```diff
@@ -1304,4 +1422,167 @@ protected boolean metadataUpperCase() {
   protected boolean batchDDL() {
     return false;
   }
+
+  protected Hash writeRefLogEntry(
+      Connection connection,
+      NamedRef ref,
+      Hash parentRefLogId,
+      Hash commitHash,
+      AdapterTypes.RefLogEntry.Operation operation,
+      long timeInMicros,
+      List<Hash> sourceHashes)
+      throws ReferenceConflictException {
+
+    Hash currentRefLogId = randomHash();
```

Comment:
Nit: Looks like we don't need to generate the `currentRefLogId` but can be moved into `buildRefLogEntry`

Reply:
I wanted to save one `Hash.of` conversion

Reply:
moved.